src/main/resources/tags/info/1.18support.tag

```diff
@@ -0,0 +1,10 @@
+type: text
+aliases: 1.18
+
+---
+
+Geyser's latest release supports 1.17.1 through to 1.18.
```

Comment:
Geyser only supports Java Edition 1.18 and Bedrock Edition 1.17.30 - 1.18 at this time.